libevmasm/Assembly.cpp

```diff
@@ -922,6 +932,84 @@ void appendBigEndianUint16(bytes& _dest, ValueT _value)
 	assertThrow(_value <= 0xFFFF, AssemblyException, "");
 	appendBigEndian(_dest, 2, static_cast<size_t>(_value));
 }
+
+uint16_t calculateMaxStackHeight(AssemblyItems const& _items, uint16_t _args)
+{
+	static auto constexpr LOC_UNVISITED = std::numeric_limits<size_t>::max();
```

Comment:
What is `LOC`?

```suggestion
	static auto constexpr UNVISITED = std::numeric_limits<size_t>::max();
```

Reply:
It's taken from evmone. It's LOCation :)